molgenis-js/src/test/java/org/molgenis/js/JsMagmaScriptEvaluatorTest.java

```diff
@@ -103,6 +103,20 @@ public void testUnitConversion()
 		assertEquals(weight, 180.7790549915996);
 	}
 
+	@Test
+	public void testReferenceMapping()
```

Comment:
I expected to see changes to script-evaluator.js so that this feature works in client-side as well.